ts/components/SelectCalendarModal.tsx

```diff
@@ -0,0 +1,128 @@
+import * as pot from "italia-ts-commons/lib/pot";
+import { Button, Container, Content, Text } from "native-base";
+import React from "react";
+import RNCalendarEvents, { Calendar } from "react-native-calendar-events";
+
+import I18n from "../i18n";
+
+type CalendarItemProps = {
+  calendar: Calendar;
+  onPress: (calendar: Calendar) => void;
```

Comment:
```suggestion
  onPress: () => void;
```

the parent component should have the responsibility of providing the `onPress` handler for this item

Reply:
Can't get this. MessageCTABar (the component opening the Modal) is already passing onCalendarSelect to the SelectCalendarModal that than pass this same function to CalendarItem using the onPress props.

Reply:
I wasn't sure of the change you requested. Going to apply the change.

Reply:
sorry I read "can't do this" instead of "can't get this" :)

Reply:
Done.